### PR TITLE
First doc pass + generated types

### PR DESCRIPTION
**Pull request by tommyjcarpenter**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,77 @@
...
+
+Creating an Index in MerQube's API with name `X` in namespace `Y`, automatically creates two securities in the SecAPI:
+1. An `index` security that holds all returns and metrics for that index
+2. An `intraday_index` security for the same, except that it will only have metrics if the index is a real time index.
+
+These securities, located on the SecAPI (`/security/index` and `/security/intraday_index` respectively), will have different IDs than the index, but have the same name `X` and namespace `Y` as the index.
+They all move in unison; if the index name or namespace is changed, the name and namespace of its linked securities also change to match.
+
+
+### Tyoes
```
Comment (aschmied) [inline README.md:61]: Types

File: README.md [hunk 2]
```diff
@@ -1,8 +1,77 @@
...
+
+MerQube makes heavy use of [pydantic](https://docs.pydantic.dev/latest/)
+
+The script `./generate_types.sh` at the root of this repo will:
+1. Pull down the latest production API (OpenAPI spec)
+1. generate a file `merqube_client_lib/pydantic_types.py
+
+We may eventually push this as a package to PyPi as well.
```
Comment (aschmied) [inline README.md:69]: If you want to be precise it's actually PyPI with a capital I

File: pylintrc [hunk 3]
```diff
@@ -12,6 +12,7 @@
 disable=C0301,  #line too long
         C0103,  #constants uppercase
         import-error, # unless youre in the venv, it won't find many packages
+        F722  # type generation induces these
```
Comment (aschmied) [inline pylintrc:15]: You might be able to use a [root template](https://koxudaxi.github.io/datamodel-code-generator/custom_template/) to add pylint annotations to the generated file. That way you wouldn't need to ignore this lint repo-wide

**Review by aschmied — APPROVED** [review]

File: pylintrc (hunk 3 above)
Comment (tommyjcarpenter) [reply pylintrc:15]: ill add that as an issue, danke

**Pull request merged**